src/main/java/culturemedia/controllers/CultureMediaController.java

```diff
@@ -0,0 +1,26 @@
+package culturemedia.controllers;
+
+import java.util.*;
+
+import culturemedia.exception.VideoNotFoundException;
+import culturemedia.model.Video;
+import culturemedia.service.Impl.CultureMediaServiceImpl;
+
+public class CultureMediaController {
+
+	private final CultureMediaServiceImpl cultureMediaService;
+
+
+	public CultureMediaController(CultureMediaServiceImpl cultureMediaService) {
+		this.cultureMediaService = cultureMediaService;
+	}
+
+
+	public List<Video> find_allVideos() throws VideoNotFoundException {
+		List<Video> videos = null;
```

Comment:
```suggestion
		List<Video> videos = cultureMediaService.findAll();
```
Darle valor a la variable videos